src/openforms/logging/logevent.py

```diff
@@ -693,3 +693,12 @@ def email_status_change(
             "include_in_daily_digest": include_in_daily_digest,
         },
     )
+
+
+# - - -
+def invalid_variable_reference_in_logic(form: Form, logic_variable: str):
+    _create_log(
+        form,
+        "invalid_variable_reference_in_logic",
+        extra_data={"logic_variable": logic_variable},
+    )
```

Comment:
we don't need DB logging for this, just a simple stdlib `logger.info(...)` is enough